container/README.md

```diff
@@ -4,17 +4,18 @@ The Dockerfile in this directory will build a docker image named `eventstore_dem
 
 **Procedure:**
 **Step 1: Build the docker image**
-User need to run the shell script `build.sh` to build the docker image.
-The image size is around 3.3 GB, build takes around 30 mins, depending on the network condition.
+Run the shell script `build.sh` to build the docker image.
+The image size is around 3.3 GB, build takes around 30 mins, depending on network conditions.
+The Event Store release the IoT applications will use must be specified. The release is used to tag the image. Supported releases are: 2.0.1.0 and 2.0.0.5.
```

Comment:
NIT - "The Event Store release **that** the IOT..."

Reply:
Also this comment is **incorrect* no ?  It doesn't have to be specified i.e. it's not mandatory, we default to "latest", can you update this comment ?

Reply:
not sure grammatically "that" is required maybe a comma but will add that that

I know there is still some code testing for es version being passed and if not latest is used. I did not pull that code as there were many bits in play. I think by requiring the version it canshould be removed.